src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -103,7 +103,7 @@ choco export
  - 0: operation was successful, no issues detected
  - -1 or 1: an error has occurred
 
-If you find other exit codes that we have not yet documented, please
+If you find other exit codes that we have not yet been documented, please

```

Comment:
@mwallner can you remove this change, as it doesn't actually make sense the way you have written it.  Once changed, we can get this merged in.